### ISSUE TITLE
Web entity does not respond to scroll bars using Macos client

### ISSUE THREAD

**Issue by ArcadeFever** [vircadia/vircadia#512]
Web entity does not respond to scroll bar using Macos client.

On my 15-inch Macbook pro running Mojave (version 10.14.6) with the 2020.2.2 release client, I get the following debug errors when trying to slide the scroll bars on web entities:

[07/05 15:54:27] [WARNING] [qt.webengine.touch] Sending simulated touch events to Chromium does not work properly on macOS. Consider using QNativeGestureEvents or QMouseEvents.

**Comment by JulianGro** [comment]
Normally you would scroll by pressing on the page and dragging it up (like on a phone). Does that work?

**Comment by ArcadeFever** [comment]
Yes, clicking on the page or on the scroll bar, and dragging should scroll the page (as it does in Windows client), but in macos client neither works.

**Comment by ArcadeFever** [comment]
After more testing, I've found some interesting observations about this bug that may help in solving the issue.

Using the Vircadia calendar located in the Hub, Here is what the scroll bar on the top right looks like (note its much thinner compared to the windows web entity scrollbar).
<img width="455" alt="1" src="https://user-images.githubusercontent.com/6512223/87868426-3b6d4080-c94a-11ea-941d-0abe4e01f608.png">

When I pass my mouse pointer over the scrollbar, it changes in width to be a little wider, I assume to indicate that its now active.
<img width="469" alt="2" src="https://user-images.githubusercontent.com/6512223/87868453-88e9ad80-c94a-11ea-811e-4a684f75b4aa.png">

If I click (and hold down left mouse button) on this bar, and keep my mouse positioned within the width of the bar, I am able to scroll the page up or down.  The moment my mouse pointer goes outside this width of this bar or I release the mouse button, the scrolling stops.  I see the same qt.webengine.touch error messages in the console while I was scrolling the page.  And now the scroll bar returns to the thinner version, and is now inactive/unclickable.
<img width="415" alt="3" src="https://user-images.githubusercontent.com/6512223/87868495-07464f80-c94b-11ea-865d-c7e238de9bea.png">

If I click on a web link within the page and get the web-entity to reload the webpage, the scroll bar becomes active again, and I am able to mouse over it and can scroll like before.  One more interesting note, in the Arcadia calendar, there is also another scrollbar within the calendar entries section of the page.  I am unable to click this or get this to scroll bar to become active.

**Comment by ArcadeFever** [comment]
FYI, browser pop up window inside interface (like when you click on the http link in chat) works perfectly fine. No problems scrolling, and no warnings in the log like with web interface.  Why is this?  Wouldn't the web interface use the same Qt browser window just mapped onto a surface?

**Comment by ArcadeFever** [comment]
Good info by HifiExperiments:
the web browser is a normal QML window that gets mouse/touch events natively, whereas the web entity is backed by QML, but we actually manually generate events to send to it based on its 3D transform and everything.  our manual generation of the events has always been wonky, especially on mac.  that code is in RenderableWebEntityItem.cpp, specifically, WebEntityRenderer::handlePointerEvent
